src/navigation/components/profile/ProfileDropdown.jsx

```diff
@@ -23,10 +24,27 @@ export const ProfileDropdownComponent = ({
 	profile,
 	logout,
 	groups,
+	groupDraft,
 	groupHome,
 	allGroupsLabel,
 	allGroupsLink,
 }) => {
+	const groupDraftsContent = groupDraft => (
+		<li
+			key={groupDraft.urlname}
+			className={cx(PROFILE_GROUP_DRAFT_LIST_ITEM_CLASS, 'list-item')}
+		>
+			<p>{groupDraft.name}</p>
+			<div>
+				<span>{groupDraft.status}</span>
+				<a href={groupHome(groupDraft.urlname)}>
+					<strong>{groupDraft.actionTitle}</strong>{' '}
+				</a>
+				<Icon className="text--blue" shape="arrow-right" size="xs" />
```

Comment:
Should the arrow be clickable as well here?

Reply:
hmm, that's a good question, I will ask Kara about that